CRM/Legacycode/Core_DAO.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/*-------------------------------------------------------+
+| SYSTOPIA - LEGACY CODE INLINE-REPLACEMENTS             |
++--------------------------------------------------------+
+| This program is released as free software under the    |
+| Affero GPL license. You can redistribute it and/or     |
+| modify it under the terms of this license which you    |
+| can read by viewing the included agpl.txt or online    |
+| at www.gnu.org/licenses/agpl.html. Removal of this     |
+| copyright header is strictly prohibited without        |
+| written permission from the original author(s).        |
++--------------------------------------------------------*/
+
+/**
+ * This class offers in-line code replacements for deprecated/dropped functions
+ *  of the CRM_Core_DAO class
+ */
+class CRM_Legacycode_Core_DAO
```

Comment:
@bjendres Or just `CRM_Legacycode_DAO`?

Reply:
> @bjendres Or just `CRM_Legacycode_DAO`?

No, I don't think that's a good idea. There a plenty more DAOs besides Core_DAO that might have static functions that might be dropped in the future. This way we can still attribute this to the original class, and for the person applying the changes it doesn't make any difference.